lib/javascript/food/foodFilter.js

```diff
@@ -0,0 +1,43 @@
+const html = require('./html')
+
+class FoodFilter {
+  constructor(){}
+
+  renderFoodFilter(){
+     $('.food-filter-container').append(html.foodFilter());
+  }
+
+  listen(){
+    $('input#food-filter').on('keyup', function(event){
+      if (event.keyCode === 8) {
+        this.unhideOnBackspace()
+      }
+      this.hideUnwantedResults()
+    }.bind(this))
+  }
+
+  hideUnwantedResults(){
+    let rows = $('.food-name')
+    $.each(rows, function(i, tdTag){
+      let foodName = (tdTag.innerText).toLowerCase();
+      let searchParams = (event.target.value).toLowerCase();
+      if (!foodName.includes(searchParams)) {
+        $(this).parent().hide()
+      };
+    });
+  }
+  unhideOnBackspace(){
```

Comment:
@charliecorrigan  & @MrAaronOlsen  this function is extremely similar to the `hideUnwantedResults` - is there a way you could avoid the repetition and make this more DRY?